examples/computer_vision/efficientdet_pytorch/startup-hook.sh

```diff
@@ -0,0 +1,16 @@
+cp -pv /run/determined/workdir/patch/__init__.py /run/determined/pythonuserbase/lib/python3.6/site-packages/determined/pytorch/__init__.py
+cp -pv /run/determined/workdir/patch/_experimental.py /run/determined/pythonuserbase/lib/python3.6/site-packages/determined/pytorch/_experimental.py
+
+cp -pv /run/determined/workdir/patch/_pytorch_trial.py /run/determined/pythonuserbase/lib/python3.6/site-packages/determined/pytorch/_pytorch_trial.py
+cp -pv /run/determined/workdir/patch/_trial_controller.py /run/determined/pythonuserbase/lib/python3.6/site-packages/determined/_trial_controller.py
+
+cp -pv /run/determined/workdir/patch/pytorch_context.py /run/determined/pythonuserbase/lib/python3.6/site-packages/determined/pytorch/_pytorch_context.py
+cp -pv /run/determined/workdir/patch/reducer.py /run/determined/pythonuserbase/lib/python3.6/site-packages/determined/pytorch/_reducer.py
```

Comment:
Assuming this is some hot patching you did that isn't necessary once the experiment is cleaned up and ready to merge?